data/operators/leisure/nature_reserve.json

```diff
@@ -156,6 +156,17 @@
         "operator:short": "USFWS",
         "operator:wikidata": "Q674113"
       }
+    },
+    {
+      "displayName": "Lake County Forest Preserves",
+      "id": "lakecountyforestpreserves-2y5i2g",
+      "tags": {
+        "leisure": "nature_reserve",
+        "operator": "Lake County Forest Preserves",
+        "operator:type": "government",
+        "operator:wikidata": "Q17068552",
+        "website": "https://www.lcfpd.org/"
```

Comment:
Please note that [we don't support `website=*` or other contact tags](https://github.com/osmlab/name-suggestion-index/wiki/Contributing#tags-required) in the NSI. I would add the website to the linked Wikidata item if it doesn't already have it.

```suggestion
```